### PR TITLE
remove all of fitbit

### PR DESCRIPTION
**Pull request by gedankenstuecke**
This should remove all 
* views
* models
* controllers
* worker integrations
* tests
* and ultimately dumps the tables that contained the data.

### CODE REVIEW

**Comment by gedankenstuecke** [comment]
Seriously, a -0.1% change in test coverage? 😂

**Comment by gedankenstuecke** [comment]
I cheated by adding a test for the `DAS` show view 😂

**Comment by gedankenstuecke** [comment]
Ready to review @tsujigiri @philippbayer

File: db/structure.sql [hunk 1]
```diff
@@ -2161,11 +2209,25 @@ INSERT INTO schema_migrations (version) VALUES ('20160626121340');
...
 INSERT INTO schema_migrations (version) VALUES ('20161226175703');
 
+INSERT INTO schema_migrations (version) VALUES ('20161226175703');
+
 INSERT INTO schema_migrations (version) VALUES ('20171113104813');
 
+INSERT INTO schema_migrations (version) VALUES ('20171113104813');
+
+INSERT INTO schema_migrations (version) VALUES ('20180118100003');
+
```
[the remark this answers is not in the document]
Comment (gedankenstuecke) [reply db/structure.sql]: I think from me running db migrate to drop the tables for fitbit. But I’m not sure why there’s the first one, the second one is from the migration I wrote.

[the remark this answers is not in the document]
Comment (tsujigiri) [reply db/structure.sql]: There is a gazillion of them, though. Maybe try this:
```sh
git checkout master -- db/structure.sql
RAILS_ENV=test bundle exec rake db:reset
RAILS_ENV=test bundle exec rake db:migrate
```
And then commit the new version, which should just be the original version plus your changes.

[the remark this answers is not in the document]
Comment (gedankenstuecke) [reply db/structure.sql]: Ah, thanks! I'll ran that 👍

**Comment by gedankenstuecke** [comment]
time to duck and cover 🏃

**Pull request merged**